### ISSUE TITLE
/notify isn't working

### ISSUE THREAD

**Issue by slackhead** [irssi/irssi#1241]
irssi 1.2.1

I added a simple:

/notify nick@host network1

but the nick shows as offline, even though he has connected and I'm now chatting to him in a /query
I then tried adding `*!*@* network1` and `nick@host` without the network, but no change.

I don't know if it makes a difference, but the network adds a custom host mask for privacy, which is a single word.  The server is running ngircd + atheme services, which I have full control over.

Any ideas?

**Comment by ailin-nemui** [comment]
I'm pretty sure the notify code is one of the sadder ones. Does it still show them as offline after the time in  /set notify_check_time  passed (by default 1min)?

**Comment by ailin-nemui** [comment]
technically Irssi is using ISON. would be great to have a more modern notify implementation...;-)

**Comment by slackhead** [comment]
Yes, they are still showing as offline after notify_check_time 

I forgot to add that the user isn't a registered with nickserv.  Would that cause a problem with ISON?

**Comment by slackhead** [comment]
/ison nick

shows him online, but /notify still doesn't.

**Comment by ailin-nemui** [comment]
does the user@host bit match, that you get from /whois Nick? 

/notify Nick!user@host

**Comment by slackhead** [comment]
Thanks, nick!user@host seems to work.

Do you know if it's possible to get it to work with the beep_beep script to make it do more than just print a message?  I'm guessing something would need adding to the beep_msg_level or perhaps I could hilight the join message.

**Comment by ailin-nemui** [comment]
you should be able to use wild cards like `nick!*@*`

you have to write a script to do anything with it

For example to add it to the beep_beep script, add this line at the end of the script:
Irssi::signal_add("notifylist joined", "beep_beep");

**Comment by slackhead** [comment]
I think I know where I was going wrong now.  When I searched online about this problem earlier, I found something that looked like an old version of the /help notify that said not to use wildcards in the nick, which I imagine is probably because /ison only takes a literal nick as argument.

I've added that signal to the end of beep_beep and reloaded it.  Thanks for that.

Should I close this now?  I ask because I was once reprimanded for closing an issue myself in a different repo(!)
